tardis/conftest.py

```diff
@@ -243,3 +252,32 @@ def simulation_verysimple_vpacket_tracking(config_verysimple, atomic_dataset):
     sim.last_no_of_packets = 4000
     sim.run_final()
     return sim
+
+
+@pytest.fixture(scope="session")
+def simulation_rpacket_tracking(config_rpacket_tracking, atomic_dataset):
+    """
+    Creating a simulation object using a simple configuration
+
+    Parameters
+    ----------
+    config_verysimple_rpacket_tracking : tardis.io.configuration.config_reader.Configuration
+    atomic_dataset : AtomData
+
+    Returns
+    -------
+    simulation object with track_rpacket enabled
+    """
+    config_rpacket_tracking.montecarlo.iterations = 3
```

Comment:
Do we need 3 iterations for the test to be useful?

Reply:
I don't think so. But it shouldn't take too much time.